src/library-authoring/common/ErrorAlert.jsx

```diff
@@ -0,0 +1,26 @@
+import { Alert, Col } from '@edx/paragon';
+import React from 'react';
+import PropTypes from 'prop-types';
+import { injectIntl } from '@edx/frontend-platform/i18n';
+import { truncateErrorMessage } from './data';
+import commonMessages from './messages';
+
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
Mostly because I have a general aversion to default exports-- I find them to be limiting and to generate confusion because there's more than one way to export something, and almost every time I use them I find myself wishing I didn't. They appear to be, as far as I can see, a holdover from CommonJS that makes little sense today.

Unfortunately the plugin we have for this expects you to prefer default exports or prefer named exports, and when I flipped the switch the other direction, I saw just how many modules I'd have to change and thought better of it. I can change it if you want me to, though.